config/routes.rb

```diff
@@ -14,6 +14,7 @@
         resources :conversations, only: :index do
           resources :messages, module: :conversations, only: %i[create index]
         end
+        resources :informations, param: :title, only: :show, path: ''
```

Comment:
Why is `path: ''` necessary?

Reply:
So the route will be: 

> /about_section 
for example, instead of 

> /informations/about_section